base_layer/wallet_ffi/Cargo.toml

```diff
@@ -17,6 +17,7 @@ tokio = "0.2.0-alpha.4"
 libc = "0.2.65"
 rand = "0.5.5"
 chrono = { version = "0.4.6", features = ["serde"]}
+get_if_addrs = "0.5.3"
```

Comment:
This has snuck back in during the merge but is not needed anymore.

Reply:
Removed in next commit